scripts/package-osx.sh

```diff
@@ -45,7 +45,7 @@ hdiutil create -volname "MusicBrainz Picard $VERSION" -srcfolder 'MusicBrainz Pi
 [ "$codesign" = '1' ] && codesign --keychain picard.keychain --verify --verbose --sign 'Developer ID Application: MetaBrainz Foundation Inc.' "$dmg"
 if [ -n "$UPLOAD_OSX" ]
 then
-    curl -v --fail --upload-file "$dmg" https://transfer.sh/
+    curl -v --upload-file "$dmg" https://transfer.sh/
```

Comment:
What about making curl command more resistant to temporary issues using `--retry`, `--retry-delay`, `--retry-max-time` options ?

Reply:
Good idea, I added `--retry 6 --retry-delay 10`. What do you think about the values? I want to give this enough time so the receiving server eventually recovers, but still don't want to delay the test run for too long.